src/unity/toolkits/drawing_classifier/drawing_classifier.mm

```diff
@@ -0,0 +1,12 @@
+#include <unity/toolkits/drawing_classifier/drawing_classifier.hpp>
+#include <unity/toolkits/drawing_classifier/mps_drawing_classifier.hpp>
+
+namespace turi {
+namespace drawing_classifier {
+
+void init_model(std::map<std::string, std::vector<float>> weights) {
+
```

Comment:
I'm confused. Seems like there should be a lot more here before it's even code complete.